Formula/jdupes.rb

```diff
@@ -0,0 +1,19 @@
+class Jdupes < Formula
+  desc "Duplicate file finder and an enhanced fork of 'fdupes'"
+  homepage "https://github.com/jbruchon/jdupes"
+  url "https://github.com/jbruchon/jdupes.git", :using => :git, :tag => "v1.9", :revision => "ba505f2a5462fc851e6d9f8a558a427eed1187c1"
```

Comment:
We prefer to use source tarballs, which allow homebrew caching mechanism to work better: `https://github.com/jbruchon/jdupes/archive/v1.9.tar.gz`